CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.8.0] - 2023-01-25
+
+### Added
+- support for Account Defender.
```

Comment:
Consider adding a reference for the User Identifiers feature name according to the Spec.
Something like - Support User Identifiers: CTS and JWT.